website/webpack.config.js

```diff
@@ -26,15 +26,6 @@ const plugins = [
   // so that code patch will never be executed.
   new webpack.IgnorePlugin(/^eslint$/, /babel-eslint/),
 
-  // Prettier //
-
-  // typescript-eslint-parser is a dev dependency of prettier, so it's not
-  // installed by default but prettier does still require it. See
-  // https://github.com/prettier/prettier/issues/986
-  new webpack.IgnorePlugin(/typescript-eslint-parser/, /\/prettier/),
```

Comment:
Ah, fixed in https://github.com/fkling/astexplorer/pull/246/commits/69276968c80cdb4f1a7c9c2eb7562043a427b3ab

Since this is a JavaScript/etc transformer, I eliminated the non-JS parsers.